README.md

```diff
@@ -64,20 +61,20 @@ our [official documentation](https://docs.conjur.org/Latest/en/Content/Developer
 
 #### Install the SDK
 
-The SDK can be installed via PyPI. Note that the SDK is a **Community** level project meaning that
-the SDK is subject to alterations that may result in breaking change.
+The SDK can be installed via PyPI. Note that the SDK is a **Community** level project meaning that the SDK is subject to
+alterations that may result in breaking change.
 
-To avoid unanticipated breaking changes, make sure that you stay up-to-date on our latest releases
-and review the project's [CHANGELOG.md](CHANGELOG.md).
+To avoid unanticipated breaking changes, make sure that you stay up-to-date on our latest releases and review the
+project's [CHANGELOG.md](CHANGELOG.md).
 
 ```
 pip3 install conjur
```

Comment:
what is the difference between this and the pip3 install in line 85?

Reply:
This one install conjur from PyPi
line 85 installs it locally from the repo